app/src/main/java/de/grobox/transportr/networks/TransportNetworks.java

```diff
@@ -635,6 +635,25 @@
 					.setGoodLineNames(true)
 					.build(),
 
+			// Nicaragua
+
+			new TransportNetworkBuilder()
+					.setId(NetworkId.MANAGUA)
+					.setName(R.string.np_name_managua)
+					.setDescription(R.string.np_desc_managua)
+					.setRegion(Region.NICARAGUA)
+					.setStatus(ALPHA)
+					.setGoodLineNames(true)
+					.build(),
+
+			new TransportNetworkBuilder()
+					.setId(NetworkId.ESTELI)
```

Comment:
This PR will probably fail CI as these `NetworkId`s are not yet included in the version of PTE included here.